cita-network/src/node_manager.rs

```diff
@@ -41,13 +39,28 @@ use tentacle::{
 pub const DEFAULT_MAX_CONNECTS: usize = 666;
 pub const DEFAULT_PORT: usize = 4000;
 pub const CHECK_CONNECTED_NODES: Duration = Duration::from_secs(3);
+
+// Score uses to manage known_nodes list. If a node has too low score, do not dial it again.
```

Comment:
Is the score system a part of p2p or we add it just to `refuse-duplicate-nodes`?

Reply:
We have to use it to fix the problem.
The problem of `refused duplicate nodes` is a little complicated:
- First, need to find a node is a `duplicate node`. I design a `init message` to exchange their address to find it.
- Second, when the server found a `duplicate node`, it will disconnected the `duplicate node`, but
- Third, the `duplicate node` will dial all the time after it was disconnected by server, so I design `score` to handle this.

Reply:
>  But for now, it useless.
So `Third` is in the todo list, right?

Reply:
Not exactly. Maybe I missed some information, the score design not just use for detecting `duplicated node`, but also `duplicated session (connection)`.
- A `duplicated session (connection)` is session connected to the same node.  For example, there are two nodes A and B, A dial to B will open a session; And B dial to A, It will open a `duplicated session` in this situation. 
- A `duplicated node` means two nodes contains the same `address` and `private key`.